node/blockchain/models/block/manager.py

```diff
@@ -23,14 +23,15 @@ def add_block_from_signed_change_request(
         signing_key: Optional[SigningKey] = None,
         validate=True
     ) -> 'Block':
-        if validate:
-            signed_change_request.validate_business_logic(blockchain_facade)
+        with TableLocker(self.model):
```

Comment:
let's use it as decorator instead to avoid excessive nesting

Reply:
When using it as decorator require import Block model. I can't import block model be course of circular import.